frontend/src/e2e-test/specs/6_mobile/child-attendances.spec.ts

```diff
@@ -509,24 +501,14 @@ describe('Child mobile attendance list', () => {
   })
 
   test('Term break child is shown in absent list', async () => {
-    const term = new FiniteDateRange(
-      LocalDate.of(2021, 8, 15),
-      LocalDate.of(2022, 5, 31)
-    )
-
-    // Term break today
-    await Fixture.preschoolTerm()
-      .with({
-        finnishPreschool: term,
-        swedishPreschool: term,
-        extendedTerm: term,
-        applicationPeriod: new FiniteDateRange(
-          LocalDate.of(2022, 1, 10),
-          LocalDate.of(2022, 1, 20)
-        ),
-        termBreaks: [new FiniteDateRange(today, today)]
-      })
-      .save()
+    // change mocked now to be during term break
+    await page.close()
+    now = HelsinkiDateTime.of(2024, 1, 1, 13, 0, 0)
+    today = now.toLocalDate()
+    page = await Page.open({ mockedTime: now })
```

Comment:
It might be a bit tedious to do, but it would be much better to refactor all tests in this suite to open the page themselves instead of having a few tests close and reopen the page with different parameters.